src/blockchain/transactions.ts

```diff
@@ -99,7 +108,7 @@ export function send(
     lastChange: new Date(),
   };
 
-  function successOrFailure(txHash: string, receipt: any): Observable<TxState> {
+  function successOrFailure(txHash: string, receipt: any, rebroadcast: TxRebroadcastStatus|undefined): Observable<TxState> {
```

Comment:
If Maciek leaves anything important , you can fix this. I think it's not formatted.
Should be like this `TxRebroadcastStatus | undefined` right?

Reply:
How to make linter complain about this?

Reply:
I will check it out ;)